scripts/lua/modules/alert_store/flow_alert_store.lua

```diff
@@ -317,10 +317,24 @@ end
 
 -- ##############################################
 
+local RNAME = {
+   ADDITIONAL_ALERTS = { name = "additional_alerts", export = true},
+   ALERT_NAME = { name = "alert_name", export = true},
+   DESCRIPTION = { name = "description", export = true},
```

Comment:
description must be present in the csv of any entity, not only on flows.

Reply:
@simonemainardi Could you be more precise about this comment, I don't understand what you mean.
Thanks

Reply:
Sure. Sorry for not being accurate enough. The full description of an alert should be exported for all alerts. For example, for host alerts, it is desirable to have exported not only the alert type label but also the full alert description. E.g.,

Not only "TCP SYN Scan" but also "devel is a SYN scan attacker [ZZ > YY SYN sent]"